bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -4,116 +4,122 @@ CE02SHBP-MJ01C-05-IP1,,Sensor,0,JBox Port J05 (LJ01D),JBox Port J05 (LJ01D),,,,,
+CGCON-BDCL11-02002,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2002,,,,DELETE?,
+CGCON-BDCL11-15004,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0004,,,,CE07SHSM-00004-DCL11 added 15000 to sn UID to distinguish it from Pioneer,Oregon State University
+CGCON-BDCL11-15005,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0005,,,,CE02SHSM-00004-DCL11,Oregon State University
+CGCON-BDCL11-15009,OL000303,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,9,,,,CE02SHSM-00001-DCL11 added 15000 to SN UID to distinguish between Pioneer and Endurance,Oregon State University
+CGCON-BDCL11-17009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-009,,,,TO BE DELETED,
+CGCON-BDCL11-50001,OL000281 R00051,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0001,,,,CP04OSSM-00004-DCL11,Woods Hole Oceanographic Institution
+CGCON-BDCL11-50002,OL000099,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0002,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-50003,OL000077,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0003,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-50004,OL000263,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0004,,,,CP03ISSM-00004-DCL11,Woods Hole Oceanographic Institution
+CGCON-BDCL11-50005,OL000231,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0005,,,,CP01CNSM-00005-DCL11,Woods Hole Oceanographic Institution
+CGCON-BDCL11-50006,OL000524,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0006,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-50007,OL000324,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0007,,,,CE02SHSM-00003-DCL11,Oregon State University
+CGCON-BDCL11-50008,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0008,,,,CE09OSSM-00005-SBD11,Oregon State University
+CGCON-BDCL11-50009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-60002,OL000594,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2016-0002,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-60701,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,60701,,,,DELET?,
+CGCON-BDCL11-70001,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-0001,,,,CP04OSSM-00006-DCL11,Woods Hole Oceanographic Institution
+CGCON-BDCL11-70009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-0009,,,,,Woods Hole Oceanographic Institution
```

Comment:
eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet shows manuf sn as 2017-009; not sure if spreadsheet or this intry in csv is incorrect.